controllers/admin/AdminImportController.php

```diff
@@ -223,6 +223,8 @@ public function __construct()
                     'id' => array('label' => $this->trans('ID', array(), 'Admin.Global')),
                     'active' => array('label' => $this->trans('Active (0/1)', array(), 'Admin.Advparameters.Feature')),
                     'name' => array('label' => $this->trans('Name', array(), 'Admin.Global')),
+                    'redirect_type' => array('label' => $this->trans('Redirect type', array(), 'Admin.Advparameters.Feature')),
+                    'id_type_redirected' =>   array('label' => $this->trans('Product ID', array(), 'Admin.Advparameters.Feature')),
```

Comment:
Be careful, this label is confusing with the ID column and should be something like "Target product / category ID".